build/patches/Add-an-always-incognito-mode.patch

```diff
@@ -683,5 +734,82 @@ new file mode 100644
 +public interface INeedSnackbarManager {
 +    void setSnackbarManager(SnackbarManager manager);
 +}
+diff --git a/chrome/common/pref_names.cc b/chrome/common/pref_names.cc
+--- a/chrome/common/pref_names.cc
++++ b/chrome/common/pref_names.cc
+@@ -3284,6 +3284,10 @@ const char kShowCaretBrowsingDialog[] =
+ const char kLacrosLaunchSwitch[] = "lacros_launch_switch";
+ #endif
+ 
++#if defined(OS_ANDROID)
++const char kAlwaysIncognitoEnabled[] = "always_incognito_enabled";
++#endif
++
+ #if BUILDFLAG(IS_CHROMEOS_ASH)
+ // String enum pref determining what should happen when a user who authenticates
+ // via a security token is removing this token. "IGNORE" - nothing happens
+diff --git a/chrome/common/pref_names.h b/chrome/common/pref_names.h
+--- a/chrome/common/pref_names.h
++++ b/chrome/common/pref_names.h
+@@ -1198,6 +1198,11 @@ extern const char kLastWhatsNewVersion[];
+ extern const char kLensRegionSearchEnabled[];
+ #endif
+ 
++#if defined(OS_ANDROID)
++#define ALWAYS_INCOGNITO_ENABLED
```

Comment:
the idea is to have a `#define` available which can be useful to me in the other patches to understand if the preference is present in the code

> I assume you need this because you are attempting non-Android builds?

no, unfortunately I haven't started yet

Reply:
It was not mentioned in the patch nor PR description, and it is an important change as we do not have patch-specific defines anywhere else.

I will remove this, I am always against any change like this which is introduced without discussion.

Reply:
> I will remove this, I am always against any change like this which is introduced without discussion.

as you want, in fact for now it is not needed.

Reply:
It is not just "as I want": this change is not mentioned in this PR nor in the patch description. Perhaps you thought it is not a big change (I do not know that, I can only assume) but it is because we do not use per-patch `#define`s anywhere.

Reply:
> It is not just "as I want"

badly expressed, but I was agreeing with you!
it is useless to add things that are not used, we always have time to do so.